src/types/native.rs

```diff
@@ -108,6 +110,9 @@ impl days_ms {
     }
 }
 
+unsafe impl Pod for days_ms {}
+unsafe impl bytemuck::Zeroable for days_ms {}
```

Comment:
There is a `#[derive]` macro to implement this without any custom unsafe code. I suggest adding a comment on why that wasn't used here, and also moving the `unsafe impl`s closer to the struct definition to make then harder to miss when making changes to the type.

Reply:
Thanks! even better. Replaced by `derive`.